org.corluma.Corluma.yml

```
@@ -0,0 +1,16 @@
+app-id: org.corluma.Corluma
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: corluma
+finish-args: [--share=network,--share=ipc,--socket=x11,--socket=wayland,--filesystem=host,--device=dri]
```

```suggestion
finish-args:
    - --share=network
    - --share=ipc
    - --socket=fallback-x11
    - --socket=wayland
    - --filesystem=host
    - --device=dri
```

Also this is a broad filesystem access. Doen't qt support using portals by default?

---

without `filesystem=host`, the app as written cannot view any directories and can only save to the home directory in my testing. It might be from using a static build, I plan to fix this and reduce the permissions scope in future release.

---

home != host.